contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -717,3 +717,46 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+/*
+* KEY COLUMN USAGE
+*/
+
+CREATE OR REPLACE VIEW information_schema_tsql.key_column_usage
+AS SELECT CAST(sys.db_name() AS sys.nvarchar(128)) AS "CONSTRAINT_CATALOG",
+    CAST((select orig_name from sys.babelfish_namespace_ext where ss.nc_nspname = nspname) AS sys.nvarchar(128)) AS "CONSTRAINT_SCHEMA",
```

Comment:
We can directly join babelfish_namespace_ext with pg_namespace_ext to avoid executing this query again. Please refer other information_schema_tsql views for similar implementation.

Reply:
Resolved in latest commit.